predicators/envs/burger.py

```diff
@@ -173,17 +178,24 @@ def _get_tasks(self, num: int,
         goal = {
             GroundAtom(self._On, [patty, bottom_bun]),
             GroundAtom(self._On, [cheese, patty]),
-            GroundAtom(self._On, [tomato, cheese]),
-            GroundAtom(self._On, [top_bun, tomato]),
+            # GroundAtom(self._On, [tomato, cheese]),
+            # GroundAtom(self._On, [top_bun, tomato]),
             GroundAtom(self._IsCooked, [patty]),
-            GroundAtom(self._IsSliced, [tomato]),
+            # GroundAtom(self._IsSliced, [tomato]),
+            # GroundAtom(self._GoalHack, [bottom_bun, patty, cheese, tomato,
+            #     top_bun])
```

Comment:
why are all these commented out for now?

Reply:
The pipeline works fine with the full goal, I just wanted to see the learned predicates/operators with something simpler so I commented those out to make the goal simpler. I'll uncomment them -- doesn't really matter but probably best to keep it as the full goal for what's on master.